tests/test_api/test_metadata.py

```diff
@@ -142,9 +162,12 @@ def test_update_01():
 def test_changesheet_array_item_nested_attributes():
     mdb = get_mongo(run_config_frozen__normal_env).db
     local_id = "sty-11-r2h77870"
+    remove_tmp_doc = False
```

Comment:
the two `remove_tmp_doc = False` additions address regressions that came up in local testing due to a difference in database state.